WINDOWS.md

```diff
@@ -0,0 +1,30 @@
+# Compiling and Installing **_libecl_** on Windows
+
+## Prerequisits:
+1. Download or clone the **_libecl_** repository to your local disk.
+
+2. Python 2.7 or 3.x installation
+   - Download a python instalation https://www.python.org/ or a python environment solution such as Anaconda https://anaconda.org/
+
+3. In command prompt, navigate to the **_libecl_** source directory you created in step 1. Use the python package manager **pip** to install **_libecl_** requirements via `pip install -r requirements.txt`
```

Comment:
Instead of _command prompt_ here we can suggest the user to open a suitable MSVC command prompt. For instance, with VS17, the entry _x64 Native Tools Command Prompt fo VS 2017_ can be found somewhere in the Start.
If we do this here, we can remove point `5` further down

Also, we could write something like "Make sure `python` can be executed and that your python environment is set up correctly (in case you are using Anaconda or any other environment manager)